utils/errors.js

```diff
@@ -0,0 +1,20 @@
+const VError = require('verror');
```

Comment:
I have never really understood the point of `verror` dependency. I am not sure this is interesting here.
But if you think so, ok.

Reply:
I think it's useful when you set a hierarchy of errors.
We may remove this from now, I am ok with that